### ISSUE TITLE
Disabled taglist fields need CSS rework

### ISSUE THREAD

**Issue by LordRazen** [octobercms/october#5671]
A disabled taglist field is not readable and seem to need a rework
```
        tags:
            label: 'mch.heads::lang.terms.tags'
            type: taglist
            customTags: false
            trigger:
                field: status
                condition: 'value[0]'
                action: disable
```
![grafik](https://user-images.githubusercontent.com/9381368/204084691-227d8a6e-aa2d-444f-a376-235b3cb13778.png)

October Version: v3.1.25

Found a similar issue with a disabled tag field in a relation: 
![grafik](https://user-images.githubusercontent.com/9381368/204128145-b79e3fd2-8580-4660-9313-8998b48974ae.png)

**Comment by daftspunk** [comment]
Hi @LordRazen 

This should be fixed in v3.1.26

**Comment by LordRazen** [comment]
Can confirm both bugs are fixed. Thanks a lot ^^
